app/Models/BoardPost.php

```diff
@@ -44,15 +44,15 @@ class BoardPost extends Model
 
     protected $guarded = [];
 
-    public function cat(){
+    public function cat() {
         return $this->belongsTo('App\Models\BoardCat', 'cat_id', 'id');
     }
 
-    public function thread(){
+    public function thread() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->